src/scss/_main.scss

```diff
@@ -582,7 +582,7 @@ nav {
 }
 
 .build-history {
-  ul {
+  .previews {
```

Comment:
this was my mistake on suggesting a generic element rule, it ended up impacting the child `ul` inside the hover tooltip. 
Using a class instead 👍